src/mds/MDSRank.cc

```diff
@@ -3566,8 +3566,6 @@ void MDSRank::create_logger()
     mdm_plb.add_u64_counter(l_mdm_capa, "cap+", "Capabilities added");
     mdm_plb.add_u64_counter(l_mdm_caps, "cap-", "Capabilities removed");
     mdm_plb.add_u64(l_mdm_heap, "heap", "Heap size");
-
-    mdm_plb.set_prio_default(PerfCountersBuilder::PRIO_DEBUGONLY);
```

Comment:
I would rather not rely on defaults if this is actually wanted by Rook. Please set it to PRIO_USEFUL and add a comment.